src/main/java/com/seailz/discordjar/model/webhook/IncomingWebhook.java

```diff
@@ -0,0 +1,198 @@
+        Checker.check(embeds.length <= 10, "There can be no greater than 10 embeds!");
+        new DiscordRequest(
+                new JSONObject()
+                        .put("embeds", new JSONArray(embeds))
+                        .put("username", usernameOverride),
+                new HashMap<>(),
+                URLS.POST.GUILDS.CHANNELS.EXECUTE_WEBHOOK.replace("{guild.id}", guildId().id()).replace("{channel.id}", channelId().id()).replace("{webhook.id}", id.id()).replace("{webhook.token}", token),
+                discordJar,
+                URLS.POST.GUILDS.CHANNELS.EXECUTE_WEBHOOK,
+                RequestMethod.POST
+        ).invoke();
+    }
+
+    /**
+     * Sends a message as the Webhook.
+     * @param message The Message object to send.
+     * @throws DiscordRequest.UnhandledDiscordAPIErrorException Thrown when an unexpected error is returned from the Discord API.
+     */
+    public void send(Message message) throws DiscordRequest.UnhandledDiscordAPIErrorException {
+        new DiscordRequest(
+                new JSONObject()
+                        .put("content", message.content()),
+                new HashMap<>(),
+                URLS.POST.GUILDS.CHANNELS.EXECUTE_WEBHOOK.replace("{guild.id}", guildId().id()).replace("{channel.id}", channelId().id()).replace("{webhook.id}", id.id()).replace("{webhook.token}", token),
+                discordJar,
+                URLS.POST.GUILDS.CHANNELS.EXECUTE_WEBHOOK,
+                RequestMethod.POST
+        ).invoke();
+    }
+
+    /**
+     * Sends a message as the Webhook.
+     * @param message The Message object to send.
+     * @param usernameOverride A username override to use when sending the message. The Webhook remains unaffected.
+     * @throws DiscordRequest.UnhandledDiscordAPIErrorException Thrown when an unexpected error is returned from the Discord API.
+     */
+    public void send(Message message, String usernameOverride) throws DiscordRequest.UnhandledDiscordAPIErrorException {
+        new DiscordRequest(
+                new JSONObject()
+                        .put("content", message.content())
+                        .put("embeds", message.embeds())
+                        .put("attachments", message.attachments())
+                        .put("username", usernameOverride),
+                new HashMap<>(),
+                URLS.POST.GUILDS.CHANNELS.EXECUTE_WEBHOOK.replace("{guild.id}", guildId().id()).replace("{channel.id}", channelId().id()).replace("{webhook.id}", id.id()).replace("{webhook.token}", token),
+                discordJar,
+                URLS.POST.GUILDS.CHANNELS.EXECUTE_WEBHOOK,
+                RequestMethod.POST
+        ).invoke();
+    }
+
+    /**
+     * Deletes this Webhook.
+     * @throws DiscordRequest.UnhandledDiscordAPIErrorException Thrown when an unexpected error is returned from the Discord API.
+     */
+    public void delete() throws DiscordRequest.UnhandledDiscordAPIErrorException {
```

Comment:
My previous comment was miss interpreted. A bot may have permission to delete a webhook in a guild - this doesn't require the webhook token.

This method should be renamed to deleteWebhookWithTokem, and then implement this endpoint https://discord.com/developers/docs/resources/webhook#delete-webhook as a method called "delete"